store/extensions/extension.json

```
@@ -0,0 +1,35 @@
+{
+  "title": "AFK",
+  "description": "Anti-AFK. An extension that tracks how long you have been AFK.",
+  "authors": [
+    {
+      "name": "danii / v19",
+      "hotel": "Origins",
+      "username": "v19",
+      "discord": "danii98",
+      "twitter": "/"
+    }
+  ],

```

- The hotel field is meant for things like .COM, .NL etc
- Leave out the twitter field if you don't have an account
```suggestion
  "authors": [
    {
      "name": "danii / v19",
      "hotel": ".COM",
      "username": "v19",
      "discord": "danii98"
    }
  ],

```